workbench-core/example/infrastructure/integration-tests/tests/datasets/accessPermissions.test.ts

```diff
@@ -315,4 +314,171 @@ describe('DataSets access permissions integration tests', () => {
       });
     });
   });
+
+  describe('removeDataSetAccessPermissions', () => {
+    it('removes a read-only access permission.', async () => {
+      const createDataSetResponse = await adminSession.resources.datasets.create({}, true);
+      const dataSetId: string = createDataSetResponse.data.id;
+      const createGroupResponse = await adminSession.resources.groups.create({}, true);
+      const { groupId } = createGroupResponse.data;
```

Comment:
could we extract this to a `beforeEach` block? It looks like its repeated in below tests

Reply:
could, but it's not repeated in every test.